src/integ_test_resources/android/amplify/integration/cdk/scripts/api.py

```diff
@@ -47,8 +48,11 @@ def _add_auth_mode(self, mode_name: str, is_default: bool, mode_config):
             self.additional_auth_modes[mode_name] = mode_config
 
     def build(self):
-        with open(self.schema_file, "r") as text_file:
-            gql_schema = text_file.read()
+        gql_schema = ""
+        for entry in os.listdir(self.schema_dir):
```

Comment:
Previously, if the `schema_file` couldn't be read, you'd get an error. Now, `os.listdir(...)` might return empty, and the `for` would pass execution. Should we add back in an explicit check to ensure `listdir()` isn't empty?

Reply:
That's a good point. I'll do that